src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/translation/PtvOutPtvInTranslator.java

```diff
@@ -412,6 +493,81 @@ private V7VmOpenApiAddressIn translateAddress(V7VmOpenApiAddress address) {
     return result;
   }
 
+  private VmOpenApiAddressStreetIn translateAddressStreet(VmOpenApiAddressStreet streetAddress) {
+    if (streetAddress == null) {
+      return null;
+    }
+    
+    VmOpenApiAddressStreetIn result = new VmOpenApiAddressStreetIn();
+    result.setAdditionalInformation(streetAddress.getAdditionalInformation());
+    result.setMunicipality(streetAddress.getMunicipality() != null ? streetAddress.getMunicipality().getCode() : null);
+    result.setPostalCode(streetAddress.getPostalCode());
+    result.setStreet(streetAddress.getStreet());
+    result.setStreetNumber(streetAddress.getStreetNumber());
+    
+    return result;
+  }
+  
+  private VmOpenApiAddressStreetWithCoordinatesIn translateAddressStreetWithCoordinates(VmOpenApiAddressStreetWithCoordinates streetAddress) {
+    if (streetAddress == null) {
+      return null;
+    }
+    
+    VmOpenApiAddressStreetWithCoordinatesIn result = new VmOpenApiAddressStreetWithCoordinatesIn();
+    result.setAdditionalInformation(streetAddress.getAdditionalInformation());
+    result.setLatitude(streetAddress.getLatitude());
+    result.setLongitude(streetAddress.getLongitude());
+    result.setMunicipality(streetAddress.getMunicipality() != null ? streetAddress.getMunicipality().getCode() : null);
+    result.setPostalCode(streetAddress.getPostalCode());
+    result.setStreet(streetAddress.getStreet());
+    result.setStreetNumber(streetAddress.getStreetNumber());
+    
+    return result;
+  }
+  
+  private VmOpenApiAddressStreetWithCoordinatesIn translateAddressStreetWithCoordinates(VmOpenApiAddressStreetWithOrder streetAddress) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "translateAddressStreetWithCoordinates" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)